web/clusters.go

```diff
@@ -41,16 +275,35 @@ func NewClusterHandler(client consul.Client) gin.HandlerFunc {
 			return
 		}
 
-		filter_query := fmt.Sprintf("Meta[\"trento-ha-cluster-id\"] == \"%s\"", clusterId)
-		hosts, err := hosts.Load(client, filter_query, nil)
+		filterQuery := fmt.Sprintf("Meta[\"trento-ha-cluster-id\"] == \"%s\"", clusterId)
+		hosts, err := hosts.Load(client, filterQuery, nil)
 		if err != nil {
 			_ = c.Error(err)
 			return
 		}
 
-		c.HTML(http.StatusOK, "cluster.html.tmpl", gin.H{
-			"Cluster": cluster,
-			"Hosts":   hosts,
+		clusterType := detectClusterType(cluster)
+		if clusterType == ClusterTypeUnknown {
+			c.HTML(http.StatusOK, "cluster_generic.html.tmpl", gin.H{
+				"Cluster": cluster,
+				"Hosts":   hosts,
+			})
+			return
+		}
+
+		nodes := NewNodes(cluster, hosts)
+
+		c.HTML(http.StatusOK, "cluster_hana.html.tmpl", gin.H{
+			"Cluster":          cluster,
+			"Nodes":            nodes,
+			"StoppedResources": stoppedResources(cluster),
+			"ClusterType":      clusterType,
+			"HealthContainer": &HealthContainer{
+				CriticalCount: nodes.CriticalCount(),
+				WarningCount:  nodes.WarningCount(),
+				PassingCount:  nodes.PassingCount(),
+				Layout:        "horizontal",
```

Comment:
Do we need the `horizontal` layout here? I thought that we wanted the vertical. If this works there is something wrong with the styles XD

Reply:
yes my bad, my code-reloader didn't get the CSS style change so it was actually vertical